github_settings.php

```diff
@@ -1,8 +1,8 @@
 <?php
-
+global $mysql_credentials_file_location, $triniate_homepage, $triniate_playpage;
 $mysql_credentials_file_location = __DIR__ . '/db_info.php';
 
-$triniate_homepage = 'http://triniate.com'; // Change this to the URL of your website
-$triniate_playpage = 'http://play.triniate.com'; // Change this to the URL where Triniate can be played
+$triniate_homepage = 'https://triniate.com/'; // Change this to the URL of your website
+$triniate_playpage = 'https://play.triniate.com/'; // Change this to the URL where Triniate can be played

```

Comment:
Please change https to http: the 3DS browser doesn't support https anymore, sadly.